src/sql/parts/dashboard/dashboardWebviewTab.component.ts

```diff
@@ -55,6 +58,22 @@ export class DashboardWebviewTab extends DashboardTab implements OnInit, IDashbo
 		return this.tab.editable;
 	}
 
+	@memoize
```

Comment:
remind me how this works? how does it handle caching?

Reply:
It only does it once for the life of the object